style.css

```diff
@@ -78,3 +84,229 @@ a {
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background: white;
+    padding-left: 10px;
+    padding-right: 10px;
+    top: 0;
+    left: 0;
+    right: 0;
+  }
+
+  .header {
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 100px;
+    height: 100vh;
+    background-image: url("images/header.png");
+  }
+
+  .headline {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 600;
+    font-size: 2em;
+  }
+
+  .headline-description {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 100;
+    font-size: 1.2em;
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
+
+  .connect {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    color: #6071fe;
+    text-transform: uppercase;
+  }
+
+  .social-links {
+    display: flex;
+    width: 50%;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 20px;
+  }
+}
+
+@media (max-width: 600px) {
+  html,
+  body {
+    max-width: 100%;
+    overflow-x: hidden;
+  }
+
+  .links,
+  ul {
+    visibility: hidden;
+  }
+
+  .hamburger {
+    display: block;
+  }
+
+  .logo {
+    font-size: medium;
+    white-space: nowrap;
+    max-width: 20%;
+  }
+
+  .navigation {
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background: white;
+    padding-left: 10px;
+    padding-right: 10px;
+    top: 0;
+    left: 0;
+    right: 0;
+  }
+
+  .header {
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 100px;
+    height: 100vh;
+    background-size: cover;
+    background-image: url("images/header.png");
+  }
+
+  .headline {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 600;
+    font-size: 2em;
+  }
+
+  .headline-description {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 100;
+    font-size: 1.2em;
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
+
+  .connect {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    color: #6071fe;
+    text-transform: uppercase;
+  }
+
+  .social-links {
+    display: flex;
+    width: 50%;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 20px;
+  }
+}
+
+@media (max-width: 960px) {
+  html,
+  body {
+    max-width: 100%;
+    overflow-x: hidden;
+  }
+
+  .links,
+  ul {
+    visibility: hidden;
+  }
+
+  .hamburger {
+    display: block;
+  }
+
+  .logo {
+    font-size: larger;
+    white-space: nowrap;
+    max-width: 20%;
+  }
+
+  .navigation {
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background: white;
+    padding-left: 10px;
+    padding-right: 10px;
+    top: 0;
+    left: 0;
+    right: 0;
+  }
+
+  .header {
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 300px;
+    height: 100vh;
```

Comment:
- [ ] Kindly, reduce the space between the headline text and your header. you can make it by reducing the `padding-top` value. It'll be better to put it at a value between 100 and 150px.

Reply:
@RolandM99 Nice find. I reduced it to 100px, let me know what you think.